tests/events/listStatusChanges.test.js

```diff
@@ -38,3 +38,38 @@ test('properties of status change', async () => {
     expect(statusChange.value.eventId).toBe(event.id);
     expect(statusChange.value.extraData).toEqual({'foo': 'bar'});
 });
+
+test('that status change contains holdToken', async () => {
+    let chartKey = testUtils.getChartKey();
+    await testUtils.createTestChart(chartKey, user.designerKey);
+    let event = await client.events.create(chartKey);
+    let holdToken = await client.holdTokens.create();
+    await client.events.hold(event.key, 'A-1', holdToken.holdToken);
+    await client.events.book(event.key, 'A-1', holdToken.holdToken);
+
+    let statusChanges = [];
+
+    for await (let statusChange of client.events.statusChanges(event.key)) {
+        statusChanges.push(statusChange);
+    }
+
+    expect(statusChanges[0].holdToken).toEqual(holdToken.holdToken);
+});
+
+test('that holdToken is null if booking without holdToken', async () => {
+    let chartKey = testUtils.getChartKey();
+    await testUtils.createTestChart(chartKey, user.designerKey);
+    let event = await client.events.create(chartKey);
+    let holdToken = await client.holdTokens.create();
+    await client.events.book(event.key, 'A-2');
+    await client.events.book(event.key, 'A-1', holdToken.holdToken);
```

Comment:
This almost feels like a bug in our API ... you shouldn't be able to pass in a hold token when booking an object that's not been held.

I'll like into what's going on server-side. For now I would remove this line from the PHP test, as it might break in the future:

await client.events.book(event.key, 'A-1', holdToken.holdToken);